QuoteVerification/dcap_quoteverify/linux/config.cpp

```diff
@@ -44,13 +44,15 @@
 
 #define MAX(x, y) (((x)>(y))?(x):(y))
 #define PATH_SEPARATOR '/'
-#define SGX_URTS_LIB_FILE_NAME "libsgx_urts.so.1"
-#define SGX_QL_QUOTE_CONFIG_LIB_FILE_NAME "libdcap_quoteprov.so.1"
-#define SGX_QL_QUOTE_CONFIG_LIB_FILE_NAME_LEGACY "libdcap_quoteprov.so"
 
+#define SGX_URTS_LIB_FILE_NAME "libsgx_urts.so.1"
 void *g_urts_handle = NULL;
 se_mutex_t g_urts_mutex;
 
+#ifndef GEN_STATIC
```

Comment:
Condition build is introduced by this macro, which means the object file cannot be reused in different build configurations. Can you also update Makefile to regenerate(or delete it first) the obj file in such case?

Reply:
Do you mean I need delete the object file after building as dynamic libs, and then regenerate the obj files for static libs building? The current situation is that object files for dynamic libs and static libs are independent of each other, you can find it [here](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/7abf1c362ef0c57c94bc53fc8b5ffc5ef3577e4d/QuoteVerification/dcap_quoteverify/linux/Makefile#L120).

Reply:
Well, I didn't notice that you added a suffix for static objects in [Makefile](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/7abf1c362ef0c57c94bc53fc8b5ffc5ef3577e4d/QuoteVerification/dcap_quoteverify/linux/Makefile#L70)
Then that's fine.